client/src/components/v2/OverlayMessage.tsx

```diff
@@ -39,7 +41,7 @@ export default function OverlayMessage() {
             value={message}
           />
           <InputGroup.Append>
-            <Button variant="outline-secondary" onClick={sendMessage}>
+            <Button variant="outline-secondary" onClick={(_) => sendMessage(message)}>
```

Comment:
Not sure how I missed this before, but I'm getting a compile error on the unused variable `_` (which can just be removed entirely)

Reply:
Just removed the underscore really quickly. Should build now!

Reply:
Generally don't commit to other people's branches unless you've communicated about it with the owner - things can get pretty messed up with multiple people working on a branch! It's one of the biggest reasons we have them at all